File: src/main/java/by/masarnovsky/releasedemon/ReleaseDemonApplication.java
package by.masarnovsky.releasedemon;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.scheduling.annotation.EnableAsync;

@SpringBootApplication
@EnableFeignClients
@EnableAsync
public class ReleaseDemonApplication {

    public static void main(String[] args) {
        SpringApplication.run(ReleaseDemonApplication.class, args);
    }
}

File: src/main/kotlin/by/masarnovsky/releasedemon/ReleaseDemonApplication.kt
package by.masarnovsky.releasedemon

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.PropertySource

@SpringBootApplication
@PropertySource("classpath:tokens.properties", "classpath:telegram.properties")
class ReleaseDemonApplication

fun main(args: Array<String>) {
	runApplication<ReleaseDemonApplication>(*args)
}
